### DISCUSSION TITLE
Change application language - android 14

### DISCUSSION THREAD

**Discussion by ZeljkoBG** [osmandapp/OsmAnd#21156] [General]
How to change the application language? That option does not exist in the app running on android 14. I found it to be changed in the Android settings. I set the Serbian Latin alphabet, but the application is Serbian Cyrillic!
I don't know how it is sometimes a mix of Cyrillic and Latin.

**Comment by ZeljkoBG** [comment]
Please someone from OsmAnd just confirm if they understood the problem?

**Comment by dmpr0** [comment]
Hi!
Starting from Android 13 the display language can only be changed in the system settings Android Settings → Apps → OsmAnd → Language

**Reply by ZeljkoBG** [reply]
Thanks for the answer. But I wrote it:
"I found it to be changed in the Android settings"

I change the language in the system to Latin and nothing. There is no Latin. Sometimes a mix with the Cyrillic alphabet appears. And in most cases (starting the application), it shows me Cyrillic instead of Latin.

**Reply by tommycrock** [reply]
But your question was how to change the language. Maybe you meant to ask if Serbian Latin is fully supported?

**Reply by ZeljkoBG** [reply]
Did you check in the application whether there is a Serbian Latin alphabet or did you come here to troll?

**Reply by tommycrock** [reply]
I came here to encourage you to clarify what you want to know, despite the attitude you give to the osmand developers.

**Comment by ZeljkoBG** [comment]
Upgrade solved the problem. Good job!

**Comment by ZeljkoBG** [comment]
**Unfortunately the problem has not been resolved. When restarting the application, it was in Cyrillic instead of Latin**

**Reply by DmitryAlexei** [reply]
On my device, only rebooting it change the app language

**Reply by ZeljkoBG** [reply]
There is no Serbian Latin on the last version (latest-night-build)!
If you think I'm doing something wrong, should I make a full video?

**Reply by DmitryAlexei** [reply]
Probably it's better to make an issue and provide clear steps to reproduce it. 
Screenshots are also appreciated

**Reply by ZeljkoBG** [reply]
https://github.com/user-attachments/assets/e3f30acc-8277-48cf-b6d5-43a035d09e84

**Reply by ZeljkoBG** [reply]
It used to be a mix of Serbian Latin and Cyrillic.
And today, on the relaunch of the application, there is a mix of English and Serbian Cyrillic.

![Screenshot_20241115_185750_OsmAnd~](https://github.com/user-attachments/assets/81c1c544-1597-40b4-9cdf-44ad3a2863e6)

**Comment by ZeljkoBG** [comment]
If I translated 6 thousand lines in Serbian Latin, should I know why the application does not see my translation?

Can I know if it's up to the app or I don't know how to set up the app?
If it's up to me, teach me. If it's up to OsmAnd, when will the bug be fixed?

For two weeks, no one has informed me or written anything.

**Reply by dmpr0** [reply]
HI! I have created an issue https://github.com/osmandapp/OsmAnd/issues/21480

**Comment by ZeljkoBG** [comment]
@DmitryAlexei
I made a video. what did you see?
